database/youtube/web-development.json

```diff
@@ -1,129 +1,138 @@
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "DesignCourse",
+    "description": "This channel will take you from zero to a fullstack developer. You will also learn UI/UX and Frontend Dev languages",
+    "url": "https://www.youtube.com/@DesignCourse/featured",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "CodeWithHarry",
+    "description": "This channel provides free Web Development course for beginners.",
+    "url": "https://www.youtube.com/@CodeWithHarry/playlists",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "hindi"
+  },
+  {
+    "name": "Bro Code",
+    "description": "This channel is dedicated to providing free coding tutorials for both beginners and pros",
+    "url": "https://www.youtube.com/@BroCodez",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "Milan Jovanović",
+    "description": "Milan is talking explicitly about .Net technologies and how to architect with it. He also shares videos about software engineering, software architecture, distributed systems, and Domain-Driven Design.",
+    "url": "https://www.youtube.com/@MilanJovanovicTech",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "Code with Ania Kubów",
+    "description": "This channel is dedicated to provide projects based on JavaScript and it's frameworks such as React, React-Native, Next.js, Node.js, Express. It also provides courses and projects related to GraphQL, databasing and more.",
+    "url": "https://www.youtube.com/@AniaKubow/featured",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "JavaScript Mastery",
+    "description": "This channel provides projects based on JavaScript and its frameworks like ReactJS, NextJS, NodeJS as well as API management. It also covers full stack projects with real life applications.",
+    "url": "https://www.youtube.com/@javascriptmastery",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "Hussein Nasser",
+    "description": "Hussein talks about various software engineering topics such as database architecture, network protocols, and more. This channel is a great source if you are interested in becoming an software engineer.",
+    "url": "https://www.youtube.com/@hnasr",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
+  {
+    "name": "Developedbyed",
+    "description": "Ed, a self-taught full stack engineer and designer from Romania, and he has been teaching code since 2018. He make learning fun and simple, covering topics such as web development, web design, UI/UX, game development, tech-related news, 3D modelling, and more!",
```

Comment:
[Typo]
```suggestion
    "name": "Developedbyed",
    "description": "Ed is a self-taught full-stack engineer and designer from Romania, and he has been teaching code since 2018. He makes learning fun and simple, covering topics such as web development, web design, UI/UX, game development, tech-related news, 3D modeling, and more!",
```

Reply:
This description was copied from youtube. @rajatnai49 could you make it short and brief? just mention what they teach :)